example/ios/Runner/Info.plist

```diff
@@ -41,5 +41,9 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+	<key>NSMicrophoneUsageDescription</key>
+	<string>Instabug needs access to microphone for bug reporting</string>
+	<key>NSPhotoLibraryUsageDescription</key>
+	<string>Instabug needs access to photo library for bug reporting</string>
```

Comment:
Can we just update the usage descriptions to match the ones in the [docs](https://docs.instabug.com/docs/flutter-integration#managing-permissions-ios-only)?

Reply:
Sure, I updated that in b16499a